scripts/add-sumar-turnos.ts

```diff
@@ -0,0 +1,43 @@
+import { Agenda } from '../modules/turnos/schemas/agenda';
+import { sumar } from '../modules/obraSocial/schemas/sumar';
+import { Types } from 'mongoose';
+
+
+async function run(done) {
+    const ultimasAgendas = {
+        // Fechas entre las que hubo problema con pacientes SUMAR
+        $and: [
+            { horaInicio: { $gte: new Date('2021-07-15T00:00:00.000-03:00') } },
+            { horaInicio: { $lte: new Date('2021-08-04T00:00:00.000-03:00') } }
+        ]
```

Comment:
Un detalle menor, pero es $and es equivalente a: 
```suggestion 
           horaInicio: { 
               $gte: new Date('2021-07-15T00:00:00.000-03:00'), 
               $lte: new Date('2021-08-04T00:00:00.000-03:00')  
          }
```